app/search/page.tsx

```
@@ -1,7 +1,189 @@
+    if (!fuse) return [];
+    const searchQuery = query.trim();
+    const results = fuse.search(searchQuery ? searchQuery : "");
+    // Further filter by book, chapter, narrator if set
+    const filtered = results
+      .map((r) => r.item)
+      .filter(
+        (item) =>
+          (!book || item.source === book) &&
+          (!chapter || item.chapter === chapter) &&
+          (!narrator ||
+            (item.narrator_name && item.narrator_name.includes(narrator))),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)